c10/util/safe_numerics.h

```diff
@@ -10,8 +10,16 @@
 #ifdef _MSC_VER
 #define C10_HAS_BUILTIN_OVERFLOW() (0)
 #include <c10/util/llvmMathExtras.h>
+#if !defined(_M_ARM64)
 #include <intrin.h>
 #else
+inline uint64_t _addcarry_u64(const uint64_t carry, const uint64_t a, const uint64_t b, uint64_t* sum)
```

Comment:
There is no need to implement `_addcarry_u64` as it is already implemented in [pytorch/safe\_numerics.h at master · pytorch/pytorch (github.com)](https://github.com/pytorch/pytorch/blob/master/c10/util/safe_numerics.h#L28) or  [fmt/format.h at master · fmtlib/fmt (github.com)](https://github.com/fmtlib/fmt/blob/master/include/fmt/format.h#L421). I'd rather improve the conditions to exclude ARM64 platform.

Reply:
To be honest https://github.com/pytorch/pytorch/commit/70fb673e51decdd8bf4e55244d910a8e5680d12f made this merge request  irrelevant as I can now compile it just fine on Windows on Arm. I am going to close it.